pkg/auto/lights/config/root.go

```diff
@@ -32,16 +34,28 @@ type Root struct {
 	OffButtons    []string `json:"offButtons,omitempty"`
 	ToggleButtons []string `json:"toggleButtons,omitempty"`
 
+	// RefreshEvery guarantees the last read state will eventually be processed if no event happens for this long
+	RefreshEvery jsontypes.Duration `json:"refreshEvery,omitempty"`
+
+	OnProcessError OnProcessError `json:"onProcessError,omitempty"`
+
 	// Now returns the current time. It's configurable for testing purposes, typically for testing the logic.
 	Now func() time.Time `json:"-"`
 }
 
+// OnProcessError if error is encountered during processing of a state, we set ttl to these defaults with option for backOff
+type OnProcessError struct {
+	// BackOffMultiplier is the wait duration multiplier for each iteration
+	BackOffMultiplier jsontypes.Duration `json:"backOffMultiplier,omitempty"`
+	MaxRetries        int                `json:"maxRetries,omitempty"`
+}
```

Comment:
The comments here are hard to understand for someone who is not familiar with the internals of the automation.
Someone who is trying to configure the automation might wonder "what is ttl?" and "for each iteration of what?"

Please describe what these config options do to the behaviour of the automation.

From what I can tell by reading the code, when the number of failed attempts reaches MaxRetries, the automation will stop retrying until the state changes again. That seems good to me; we need a comment saying what the behavior is supposed to be.